Rocket.Chat/Extensions/UINavigationBar/UINavigationItem.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  UINavigationItem.swift
+//  Rocket.Chat
+//
+//  Created by Rudrank Riyam on 29/03/19.
+//  Copyright © 2019 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+extension UINavigationItem {
+    func moreButton() {
```

Comment:
Can you think of a better descriptive name for this method?

Like `setupMoreButtonAccessibility()`

Reply:
I changed it to `applyMoreButtonAccessibility()`